daemon/memcached.c

```diff
@@ -7371,6 +7372,11 @@ int main (int argc, char **argv) {
                     strerror(errno));
             exit(EX_OSERR);
         }
+        if (prctl(PR_SET_DUMPABLE, 1) < 0) {
+            settings.extensions.logger->log(EXTENSION_LOG_WARNING, NULL,
+                    "failed to set dumpable flag to ensure core dumps are possible");
```

Comment:
Should we be printing out the errno and/or string representation of that errno (like on line 7372)?